spec/simple_scheduler/at_spec.rb

```diff
@@ -1,6 +1,36 @@
 require "rails_helper"
 
 describe SimpleScheduler::At, type: :model do
+  describe "AT_PATTERN" do
+    let(:pattern) { SimpleScheduler::At::AT_PATTERN }
+
+    it "matches valid times" do
+      match = pattern.match("0:00")
+      expect(match[2]).to eq("0")
+      expect(match[3]).to eq("00")
+
+      match = pattern.match("9:30")
+      expect(match[2]).to eq("9")
+      expect(match[3]).to eq("30")
+
+      match = pattern.match("Sat 23:59")
+      expect(match[2]).to eq("23")
+      expect(match[3]).to eq("59")
+
+      match = pattern.match("Sun 00:00")
+      expect(match[2]).to eq("00")
+      expect(match[3]).to eq("00")
+    end
+
+    it "doesn't match invalid times" do
+      expect(pattern.match("99:99")).to eq(nil)
```

Comment:
Add a test here for `12:0`, because a single 0 here was valid before this change.